api/handler.js

```diff
@@ -14,6 +14,98 @@ var handler = {
     }
   },
 
+// -------------------------------------------------- \\
+
+
+  findJobs : function(request, reply) {
+    if (request.auth.isAuthenticated) {
```

Comment:
do we need all these auth checks if we've already applied the authentication strategy to all routes by default?